docs/project/development-guide.md

```diff
@@ -14,9 +14,9 @@
     - [Pre-commit Hooks](#pre-commit-hooks)
     - [Signing off commits](#signing-off-commits)
     - [Incorporating upstream changes from master](#incorporating-upstream-changes-from-master)
-  - [Feast Python SDK / CLI](#feast-python-sdk--cli)
```

Comment:
can you do a code search to verify there are no references to the links you changed?

Reply:
I thought I had caught them all with `grep -R`, but I found one more ref in the .github folder. Just amended the last commit. All of the link refs should be updated now.